src/LoggedInWindow.xaml

```diff
@@ -2,145 +2,85 @@
-                                    <Setter Property="Background" Value="{StaticResource SecondaryBrush}"/>
-                                    <Setter Property="Foreground" Value="{StaticResource PrimaryBrush}"/>
-                                    <Setter Property="RenderTransformOrigin" Value="0.5,0.5"/>
-                                    <Setter Property="RenderTransform">
-                                        <Setter.Value>
-                                            <ScaleTransform ScaleX="1.05" ScaleY="1.05"/>
-                                        </Setter.Value>
-                                    </Setter>
-                                </Trigger>
-                                <Trigger Property="IsPressed" Value="True">
-                                    <Setter Property="Background" Value="{StaticResource PrimaryBrush}"/>
-                                    <Setter Property="Foreground" Value="{StaticResource TertiaryBrush}"/>
-                                </Trigger>
-                            </Style.Triggers>
-                        </Style>
-                    </Button.Style>
-                    <Button.Effect>
-                        <DropShadowEffect ShadowDepth="5" Color="#0F4C75" BlurRadius="10" Opacity="0.6"/>
-                    </Button.Effect>
+                <!-- My Lists Button -->
+                <StackPanel Style="{StaticResource HoverableButton}" Orientation="Horizontal" Margin="10" Cursor="Hand" HorizontalAlignment="Stretch" MouseLeftButtonUp="MyListsButton_Click">
+                    <Image Source="pack://application:,,,/Assets/Icons/lists_icon_white.png" 
+                           Height="30" Width="30" VerticalAlignment="Center" />
+                    <TextBlock Name="MyListsButton" Margin="10,0,0,0" VerticalAlignment="Center" 
+                               Foreground="{StaticResource SecondaryBrushB}" FontSize="20" FontWeight="Bold" 
+                               Text="Lists" TextTrimming="CharacterEllipsis"/>
+                </StackPanel>
+            </StackPanel>
+
+            <!-- Bottom Section with Logo and Logout Button -->
+            <StackPanel Grid.Row="2" Margin="5">
+                <Image Source="pack://application:,,,/Assets/Logos/Cupboard_Logo_darkmode_1.png" 
+                       Height="200"
+                       Width="200"
+                       HorizontalAlignment="Center" 
+                       Margin="0,0,0,0"
+                       Opacity="0.8" />
+
+                <Button Style="{StaticResource ExpandButtonStyle}" 
+                    Background="Red" 
+                    BorderBrush="Transparent" 
+                    BorderThickness="1" 
+                    HorizontalAlignment="Stretch" 
+                    VerticalAlignment="Center" 
+                    Padding="10" 
+                    Margin="0, 0, 0, 20"
+                    MouseLeftButtonUp="LogoutButton_Click">
```

Comment:
Logout button is doing no action right now. Add the function in the LoggedInWindow.xaml.cs and handle log out